components/brave_wallet/browser/simulation_service.h

```diff
@@ -72,11 +75,22 @@ class SimulationService : public KeyedService, public mojom::SimulationService {
                             const std::string& user_account,
                             APIRequestResult api_request_result);
 
+  void OnGetLatestSolanaBlockhash(
+      mojom::SolanaTransactionRequestUnionPtr request,
+      const std::string& chain_id,
+      const std::string& language,
+      ScanSolanaTransactionCallback callback,
+      const std::string& latest_blockhash,
+      uint64_t last_valid_block_height,
+      mojom::SolanaProviderError error,
+      const std::string& error_message);
+
   void OnScanSolanaTransaction(ScanSolanaTransactionCallback callback,
                                const std::string& user_account,
                                APIRequestResult api_request_result);
 
   api_request_helper::APIRequestHelper api_request_helper_;
+  raw_ptr<JsonRpcService> json_rpc_service_ = nullptr;  // NOT OWNED
```

Comment:
nit: raw_ptr already means that this is non-owning pointer.

That form still appears in code only because of such mechanical refactoring:
`JsonRpcService* json_rpc_service_ = nullptr;  // NOT OWNED`
VVVV
`raw_ptr<JsonRpcService> json_rpc_service_ = nullptr;  // NOT OWNED`